[result_ownership.py]
  [test_throw_exception[root-True\]]
    expected: ERROR

  [test_throw_exception[none-False\]]
    expected: ERROR

  [test_throw_exception[None-False\]]
    expected: ERROR

  [test_invalid_script[root-True\]]
    expected: ERROR

  [test_invalid_script[none-False\]]
    expected: ERROR

  [test_invalid_script[None-False\]]
    expected: ERROR

  [test_rejected_promise[root-True\]]
    expected: ERROR

  [test_rejected_promise[none-False\]]
    expected: ERROR

  [test_rejected_promise[None-False\]]
    expected: ERROR

  [test_return_value[root-True-True\]]
    expected: ERROR

  [test_return_value[root-True-False\]]
    expected: ERROR

  [test_return_value[none-False-True\]]
    expected: ERROR

  [test_return_value[none-False-False\]]
    expected: ERROR

  [test_return_value[None-False-True\]]
    expected: ERROR

  [test_return_value[None-False-False\]]
    expected: ERROR
